src/main/java/uk/nhs/cdss/config/FhirConfig.java

```diff
@@ -16,8 +25,22 @@ public FhirContext fhirContext() {
     return fhirContext;
   }
 
-  @Bean
-  public IParser fhirParser() {
-    return fhirContext().newJsonParser();
+  @PostConstruct
+  private void configureClientInterceptors() {
```

Comment:
this is a fine approach just wondering why you decided not to use the common authenticated client factory here?

Reply:
Ah, that's a good point, I haven't actually configured any client authentication at all as it's not used in the DOS.

This change to move the interceptors to the FHIR config rather than the AuditConfig is in line with the changes in the EMS where AuditConfig conflicted with the existing interceptor config, but here there are no FHIR clients at all it's redundant.